recipes/e3sm_diags/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "e3sm_diags" %}
+{% set version = "2.2.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/e3sm_diags/archive/v{{ version }}.tar.gz
+  sha256: e1211d089a7857d11114071f38f099f25f3e6bb6d69d6c6e4e2cc1ccbb7b0dc0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - e3sm_diags = acme_diags.acme_diags_driver:main
+    - acme_diags = acme_diags.acme_diags_driver:main
+    - e3sm_diags_vars = acme_diags.acme_diags_vars:main
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
```

Comment:
You probably just need pip, as setuptools is a dep

Reply:
Thanks, that makes sense. I’ve removed it.